pkg/providers/imagefamily/k8sversion.go

```diff
@@ -0,0 +1,25 @@
+/*
+Portions Copyright (c) Microsoft Corporation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package imagefamily
+
+import (
+	"context"
+)
+
+type K8sVersionProvider interface {
```

Comment:
minor: Same comment, `Kubernetes` rather than k8s IMO, and same with the function (`Kubernetes` rather than `Kube`)

Reply:
I updated the provider, but the func is a remnant from before with other existing code deps. I can address the func name in the follow up PR that will be doing some refactoring for the usage of these statuses.